test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -0,0 +1,129 @@
+﻿function Write-LogPassThru
+{
+    Param(
+        [Parameter(Mandatory=$true, ValueFromPipeline=$true, Position = 0)]
+        [string] $Message,
+        $Path = "$env:Temp\CodeCoverageRunLogs.txt"
+    )
+
+    $message = "$([Datetime]::Now.ToShortDateString()) - $([datetime]::Now.ToShortTimeString()) : $message"
+
+    if(-not (Test-Path $Path))
+    {
+        Set-Content -Path $Path -Value $Message -Force -PassThru
+    }
+    else
+    {
+        Add-Content -Path $Path -Value $Message -PassThru
+    }
+}
+
+$codeCoverageZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/CodeCoverage.zip'
+$testContentZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/tests.zip'
+$openCoverZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/OpenCover.zip'
+
+Write-LogPassThru -Message "codeCoverageZip: $codeCoverageZip"
+Write-LogPassThru -Message "testcontentZip: $testContentZip"
+Write-LogPassThru -Message "openCoverZip: $openCoverZip"
+
+$outputBaseFolder = "$env:Temp\CC"
+$null = New-Item -ItemType Directory -Path $outputBaseFolder -Force
+
+$openCoverPath = "$outputBaseFolder\OpenCover"
+$testPath = "$outputBaseFolder\tests"
+$psBinPath = "$outputBaseFolder\PSCodeCoverage"
+$openCoverTargetDirectory = "$outputBaseFolder\OpenCoverToolset"
+$outputLog = "$outputBaseFolder\CodeCoverageOutput.xml"
+$psCodeZip = "$outputBaseFolder\PSCode.zip"
+$psCodePath = "$outputBaseFolder\PSCode"
+
+try
+{
+    Write-LogPassThru -Message "Starting downloads."
+
+    Invoke-WebRequest -uri $codeCoverageZip -outfile "$outputBaseFolder\PSCodeCoverage.zip"
+    Invoke-WebRequest $testContentZip -outfile "$outputBaseFolder\tests.zip"
+    Invoke-WebRequest $openCoverZip -outfile "$outputBaseFolder\OpenCover.zip"
```

Comment:
Fixed.

---
In reply to: [94474590](https://github.com/PowerShell/PowerShell/pull/2918#discussion_r94474590) [](ancestors = 94474590)